packages/roosterjs-editor-dom/lib/list/VList.ts

```diff
@@ -330,15 +330,11 @@ export default class VList {
      */
     getListItemIndex(input: Node) {
```

Comment:
I can understand this change, but now this function makes people confusing. Need some comment to explain how it works for mixed list.